lib/chef/knife/vra_catalog_list.rb

```diff
@@ -36,6 +36,10 @@ class VraCatalogList < ResourceListCommand
           include VraServiceHelpers
         end
 
+        option :project_id,
+          long:        "--project-id",
+          description: "Project ID to fetch catalogs from."
```

Comment:
can we come up with a `description` like Catalogs are retrieved using the project ID.
Just a only suggestion

Reply:
Done